inc/stream/namespace.php

```diff
@@ -88,3 +89,19 @@ function override_network_admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
 		]
 	);
 }
+
+/**
+ * Add the Xray ID to the log item meta data.
+ *
+ * @param array $record The stream log record to filter.
+ *
+ * @return array
+ */
+function filter_wp_stream_record_array( $record ) : array {
+	if ( ! function_exists( 'XRay\\get_root_trace_id' ) ) {
```

Comment:
I think `function_exists()` requires the fully qualified function name eg. `HM\\Platform\\Xray\\get_root_trace_id`. Did you check this with xray switched off via your config? Set the property at the path `extra.altis.modules.cloud.xray` to false in your `composer.json` to confirm it.

Reply:
ah yes good point